source/code/include/playfab/PlayFabApiSettings.h

```diff
@@ -18,7 +18,7 @@ namespace PlayFab
         std::string developerSecretKey; // Developer secret key. These keys can be used in development environments.
 
     public:
-        static const std::map<std::string, std::string> requestGetParams;
```

Comment:
>static const [](start = 8, length = 12)

could this possibly break users if they utilized requestGetParams?

Reply:
Yes and no.
I already moved it from static PlayFabSettings, to PlayFabApiSettings.
The intent was to remove the static then, but somehow I missed it.
So, THIS static removal isn't going to break players, but the previous checkin might.
Either way, there's no real reason for them to use it, since it only defines extra headers for the client to send PlayFab, and the only legit usage is internal atm.